.nvmrc

```
@@ -0,0 +1 @@
+lts/*
```

The [contributing guide](https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md) recommends Node v17, this should probably be enforced here.
```suggestion
v17
```